readme.md

```diff
@@ -94,7 +125,7 @@ How is the concept of OAuth related to a valet key?
 
 Your Answer:
 ```text
-
+  There is one key for each car and if you lose the key you lose your job.  
```

Comment:
You're right.  That is exactly how OAuth works. ;)
If you don't see the connection (which is understandable), then at least share your knowledge of OAuth.